src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -55,6 +55,12 @@
     @Value("${definition.cache.ttl.secs}")
     private String definitionCacheTTLSecs;
 
+    @Value("${async.core.pool.size}")
+    private String asyncCorePoolSize;
+
+    @Value("${async.max.pool.size}")
+    private String asyncMaxPoolSize;
+
```

Comment:
May I ask why Not have Integer here

Reply:
was not aware Spring can handle automatically conversion from String to Integer. Will change this cheers